extension/js/common/core/mime.ts

```diff
@@ -343,19 +343,20 @@ export class Mime {
   }
 
   private static getNodeContentAsUtfStr = (node: MimeParserNode): string => {
+    let resultBuf = Buf.fromRawBytesStr(node.rawContent!);
```

Comment:
The solution looks good but it appears you are decoding the result twice in some situations, which is not necessary. Once here, and once again if any of the IFs fit. I would follow the original pattern to avoid unnecessary double decoding.

Reply:
Yes, thanks,

I will change code in another way so it won't double decode that.